src/actions/breakpoints/tests/syncing.js

```diff
@@ -235,4 +235,59 @@ describe("reloading debuggee", () => {
     expect(findScopeByName).toHaveBeenCalled();
     expect(update).toMatchSnapshot();
   });
+  it("syncs with changed source and an existing disabled BP", async () => {
```

Comment:
Maybe put these two tests inside their own describe in order to separate them, put the warning message in the describe and avoid repeating the complicated test description.